<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Admin_create_new_staff</class>
 <widget class="QDialog" name="Admin_create_new_staff">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1107</width>
    <height>846</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ClosedHandCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>SignUp Page</string>
  </property>
  <property name="windowIcon">
   <iconset theme="Icon">
    <normaloff>D:/education/Term/database/PyQt5-Project-Test</normaloff>D:/education/Term/database/PyQt5-Project-Test</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(199, 233, 255);</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QLabel" name="create_new_staff_header_L">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>30</y>
     <width>471</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ClosedHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 102, 104);</string>
   </property>
   <property name="text">
    <string>Create new Staff</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="create_new_staff_Firstname_L">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>200</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 85, 127);
color: rgb(245, 243, 255);</string>
   </property>
   <property name="text">
    <string>First name</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="create_new_staff_Family_L">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>200</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 85, 127);
color: rgb(245, 243, 255);</string>
   </property>
   <property name="text">
    <string>Family</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="create_new_staff_Firstname_LE">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>200</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
  </widget>
  <widget class="QPushButton" name="Signup_Button">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>710</y>
     <width>251</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(35, 255, 97);</string>
   </property>
   <property name="text">
    <string>Create</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="create_new_staff_Family_LE">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>200</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
  </widget>
  <widget class="QLabel" name="create_new_staff_Role_L">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>290</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 85, 127);
color: rgb(245, 243, 255);</string>
   </property>
   <property name="text">
    <string>Role</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="create_new_staff_StartDate_L">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>450</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ClosedHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(7, 85, 127);
color: rgb(255, 249, 249);</string>
   </property>
   <property name="text">
    <string>Start Date</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QRadioButton" name="create_new_staff_Recep_RD">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>290</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 111, 111);</string>
   </property>
   <property name="text">
    <string>Receptionist</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="create_new_staff_Housekeeper_RD">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>290</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 111, 111);</string>
   </property>
   <property name="text">
    <string>Housekeeper</string>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="create_new_staff_National_Id_L">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>290</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 85, 127);
color: rgb(245, 243, 255);</string>
   </property>
   <property name="text">
    <string>National Id</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="create_new_staff_Salary_L">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>570</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(7, 85, 127);
color: rgb(255, 249, 249);</string>
   </property>
   <property name="text">
    <string>Salary</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="create_new_staff_National_Id_LE">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>290</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
  </widget>
  <widget class="QLineEdit" name="Address_LE">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>570</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
  </widget>
  <widget class="QDateEdit" name="create_new_staff_startDate_DE">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>460</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(231, 255, 248);</string>
   </property>
  </widget>
  <widget class="QLabel" name="create_new_staff_Email_L">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>370</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 85, 127);
color: rgb(245, 243, 255);</string>
   </property>
   <property name="text">
    <string>Email</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="create_new_staff_Email_LE">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>370</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
  </widget>
  <widget class="QLabel" name="create_new_staff_Password_L">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>370</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 85, 127);
color: rgb(245, 243, 255);</string>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="create_new_staff_Password_LE">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>360</y>
     <width>271</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
